tests/integration/rooms.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { setup, clearDb, get, post } from '../utils';
+import { getRepository } from 'typeorm';
+import * as HttpStatus from 'http-status-codes';
+
+import { Room } from '../../src/entity/Room';
+
+describe('Rooms Routes', () => {
+  beforeAll(setup);
+  beforeEach(clearDb);
+
+  describe('GET /rooms/:id', () => {
+    it('should return a room', async () => {
+      const room = await getRepository(Room).save({
```

Comment:
nit: don't mix an await call with a method call to the unboxed object in the same line. Call ```save``` in a separate object.

Reply:
I don't see the problem (on this line at least), if we are not planning on reusing the value returned by `getRepository` I don't think it's bad to chain the function calls, typescript will yell at us if we are calling await to a non-promised value :)

Reply:
Yes, this is extremely **nit**. I don't like it but there isn't a problem or nothing bad at all.   
My call was because I always tend to give ```await``` calls space to be surrounded by ```try catch``` blocks.